lightning-tag/implementation-google-ad-manager.md

```diff
@@ -113,26 +126,28 @@ This example uses `lotame` as the Google audience targeting key and `lpid` as th
       // Callback when targeting audience is ready to push latest audience data
       var audienceReadyCallback = function (profile) {
 
-        // Get audiences in a comma-separated string which conforms to Google Ads input format
-        var lotameAudiences = profile.getAudienceString(',') || '';
+        // Get audiences as an array
+        var lotameAudiences = profile.getAudiences() || [];
     
         // Set the new target audiences for call to Google
         googletag.cmd.push(function() {
-          window.googletag.pubads().setTargeting('lotame', lotameAudiences);
+          window.googletag.pubads().setTargeting(audTargetingKey, lotameAudiences);
         });  
 
         var lotamePid = profile.getProfileId() || '';
-        googletag.cmd.push(function() {
-          window.googletag.pubads().setTargeting('lpid', lotamePid);
-        });
+        if (lotamePid) {
+          googletag.cmd.push(function() {
+            window.googletag.pubads().setTargeting(pidTargetingKey, lotamePid);
+          });
+        }
       };
     
       // Lotame Config
       var lotameTagInput = {
         data: {},
         config: {
-          clientId: <lotameClientId>,
-          audienceLocalStorage: true, // written to 'lotame_<lotameClientId>_auds' key
+          clientId: Number(lotameClientId),
+          audienceLocalStorage: audLocalStorageKey,
```

Comment:
looks like these were already here/added here instead of the original place in data-collection.md ?  so we're suggesting a different point of entry for clients ?

Reply:
Correct, slightly improving the syntax so they set their values up once at top of script instead of having to make sure they got all instances.